packages/server/src/index.ts

```diff
@@ -1453,7 +1453,7 @@ export class App {
                 chatflowid,
                 chatType,
                 chatId,
-                memoryType: memoryType ?? (chatId ? IsNull() : undefined),
+                memoryType: memoryType ?? undefined,
```

Comment:
couldn't remember why we have the `isNull()` in the first place, but would you be able to check from the View Message dialog on UI, that it still works as expected

Reply:
It works as expected.
https://github.com/FlowiseAI/Flowise/assets/10228950/481bf246-e62a-4ec9-b91f-7ac73fd8c6b5

I found a bug with the way the chatType is filtered. I'll add another commit to this PR to resolve that issue as well.